assignment/tool/src/webapp/vm/assignment/assignment_macros.vm

```diff
@@ -411,24 +411,25 @@
         #set($dynWidth = 100.0 / $!progressStatus.size())
         #set($totalWidthBg = 0)
         #set($statusSet = $!progressStatus.keySet())
-        <div class="progress-container">
+        #set($currentStatus = $tlang.getString("gen.notsta"))
+        <div tabindex="0" class="progress-container" role="status" aria-labelledby="status">
```

Comment:
So the container can now be focused? What can be done with the focus?

Reply:
Yes, it's meant for the screen reader.

"Now you’re able to select the progres bar with the tab key, when using a screen reader it will tell you the current status [ not started | in progress | submitted | returned ] followed by “status“ as the selected element role." - [Jira comment](https://sakaiproject.atlassian.net/browse/SAK-43400?focusedCommentId=202192)

Should we use another method?